roles/matrix-prometheus/defaults/main.yml

```diff
@@ -26,6 +26,13 @@ matrix_prometheus_systemd_wanted_services_list: []
 # Takes an "<ip>:<port>" or "<port>" value (e.g. "127.0.0.1:9090"), or empty string to not expose.
 matrix_prometheus_container_http_host_bind_port: ''
 
+# A list of extra arguments to pass to the prometheus process
+matrix_prometheus_process_extra_arguments:
+  - "--config.file=/etc/prometheus/prometheus.yml"
+  - "--storage.tsdb.path=/prometheus"
+  - "--web.console.libraries=/usr/share/prometheus/console_libraries"
+  - "--web.console.templates=/usr/share/prometheus/consoles"
```

Comment:
Are these some defaults use by the Prometheus image already?

Shouldn't we use `matrix_prometheus_process_extra_arguments: []` as a default.

Reply:
Yes, that is is the default [CMD of prometheus' dockerfile](https://github.com/prometheus/prometheus/blob/main/Dockerfile#L25).

As it's `CMD`, I'm not sure just appending custom args is a good idea, because (IIRC) `CMD` will be re-defined with any args you pass after the image in the `docker run` command

Reply:
Maybe we should have a new variable `matrix_prometheus_process_arguments: "{{ matrix_prometheus_process_default_arguments + matrix_prometheus_process_extra_arguments }}`.

And we can put the defaults in `matrix_prometheus_process_default_arguments` and allow people to add additions to `matrix_prometheus_process_extra_arguments`. If someone needs to override the defaults as well, they can manipulate `matrix_prometheus_process_default_arguments` or even `matrix_prometheus_process_arguments`

Reply:
ok, added